presentation/backend/src/presentation-backend/PresentationManager.ts

```diff
@@ -268,12 +283,26 @@ export interface PresentationManagerProps {
   caching?: {
```

Comment:
@grigasp documentation generator can't see documentation of properties of inline-declared interfaces.
```
Invalid Link: "[PresentationManagerProps.caching.workerConnectionCacheSize]($presentation-backend)" in file "presentation/advanced/caching"
Invalid Link: "[PresentationManagerProps.caching.hierarchies]($presentation-backend)" in file "presentation/advanced/caching"
```
I will remove the offending doc links for now. Please follow up with a separate PR to declare a named interface.
See #4427 for similar situation.

Reply:
Thanks. Fixing with https://github.com/iTwin/itwinjs-core/pull/4457